apps/etterlatte-saksbehandling-ui/client/src/components/person/saksoversikt.tsx

```diff
@@ -6,59 +7,108 @@ import { Heading, Tag } from '@navikt/ds-react'
 import { ManueltOpphoerModal } from './ManueltOpphoerModal'
 import { ToKolonner } from '../toKolonner/ToKolonner'
 import { Grunnlagshendelser } from './grunnlagshendelser/Grunnlagsendringshendelser'
-import { tagColors } from "~shared/Tags";
-import { formaterEnumTilLesbarString } from "~utils/formattering";
-
-export const Saksoversikt = ({
-  behandlingliste,
-  grunnlagshendelser,
-}: {
-  behandlingliste: IBehandlingsammendrag[] | undefined
-  grunnlagshendelser: Grunnlagsendringshendelse[] | undefined
-}) => {
+import { tagColors } from '~shared/Tags'
+import { formaterEnumTilLesbarString } from '~utils/formattering'
+import { hentBehandlingerForPerson, hentGrunnlagsendringshendelserForPerson } from '~shared/api/behandling'
+import { Container } from '~shared/styled'
+import Spinner from '~shared/Spinner'
+
+export const Saksoversikt = ({ fnr }: { fnr: string | undefined }) => {
+  const [behandlingliste, setBehandlingliste] = useState<IBehandlingsammendrag[] | undefined>([])
+  const [grunnlagshendelser, setGrunnlagshendelser] = useState<Grunnlagsendringshendelse[] | undefined>([])
+  const [lastet, setLastet] = useState<boolean>(false)
+  const [behandlinglisteError, setBehandlinglisteError] = useState<IBehandlingsammendrag[]>()
+  const [grunnlagshendelserError, setGrunnlagshendelserError] = useState<Grunnlagsendringshendelse[]>()
+
   const behandlinger = behandlingliste ? behandlingliste : []
   const sakId = behandlinger[0]?.sak
 
   const sortertListe = behandlinger.sort((a, b) =>
     new Date(b.behandlingOpprettet!) > new Date(a.behandlingOpprettet!) ? 1 : -1
   )
 
+  useEffect(() => {
```

Comment:
Vi har laget en `useApiCall`-hjelpefunksjon vi ønsker å bruke til api-kallene. Da slipper vi denne boilerplate-koden (ser for øvrig at `useApiCall` ikke var brukt fra før i Person-fila, så ikke lett for deg å vite det). Sjekk ut feks `Virkningstidspunkt.tsx` for et eksempel på bruk 😊

Reply:
https://jira.adeo.no/browse/EY-1534